contracts/asset-proxy/contracts/src/interfaces/IERC20Bridge.sol

```diff
@@ -0,0 +1,40 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+
+
+contract IERC20Bridge {
+
+    /// @dev Transfers `amount` of the ERC20 `tokenAddress` from `from` to `to`.
+    /// @param bridgeData Arbitrary asset data needed by the bridge contract.
+    /// @param tokenAddress The address of the ERC20 token to transfer.
+    /// @param from Address to transfer asset from.
+    /// @param to Address to transfer asset to.
+    /// @param amount Amount of asset to transfer.
+    /// @return success The magic bytes `0xb5d40d78` if successful.
+    function transfer(
```

Comment:
Some nits about the signature of this function:

`transferFrom` is more descriptive than `transfer`. It's somewhat awkward that this would be the 3rd different `transferFrom` used in the proxies, so maybe there is a better name... but I don't love `transfer`.

I also think `bridgeData` should be the last parameter. It is optional and probably won't be used in all bridge contracts. Making it the last parameter is consistent with all of the token standards that have a similar parameter in their own transfer functions.

Reply:
I would like to avoid `transferFrom` because it makes it really hard to explain this pattern to people (`transferFrom` calls `transferFrom` calls `transferFrom`). Maybe `convertFrom`? `exchangeFrom`? `bridgeFrom`? `swapFrom`? `settleFrom`? I'm at a loss.

Reply:
`withdrawTo` perhaps?